missing dependency in conda environment

I tried to build the portal site locally. From the repository root:
```
conda env create -f ci/environment.yml
conda activate
make html
```
generated the following error:
```
Extension error:
Could not import extension gallery_generator (exception: No module named 'pandas')
make: *** [html] Error 2
```
This was quickly fixed by doing `conda install pandas`.

I will submit a PR to add pandas to `environment.yml`

--- a/ci/environment.yml
+++ b/ci/environment.yml
@@ -9,3 +9,4 @@ dependencies:
   - sphinx-autobuild
   - matplotlib
   - sphinx-book-theme
+  - pandas

